GetDelegatesByOrganization/handler.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  IResponseErrorInternal,
+  IResponseErrorNotFound,
+  IResponseSuccessJson,
+  ResponseErrorInternal
+} from "@pagopa/ts-commons/lib/responses";
+import { wrapRequestHandler } from "@pagopa/ts-commons/lib/request_middleware";
+import * as express from "express";
+import { pipe } from "fp-ts/lib/function";
+import * as TE from "fp-ts/lib/TaskEither";
+import { OrganizationDelegates } from "../generated/definitions/OrganizationDelegates";
+
+type GetDelegatesByOrganizationResponseHandler = () => Promise<
+  | IResponseSuccessJson<{ readonly data: OrganizationDelegates }>
```

Comment:
Why `{ data: OrganizationDelegates }` and not just `OrganizationDelegates`?

As per specification, we don't expect a `data` field: https://github.com/pagopa/io-subscription-migration/blob/IC-267-Handler-Setup/openapi/index.yaml#L57

Reply:
Because `OrganizationDelegates` is an `array` and when I'll return a **ResponseSuccessJson** with an Array I get something like:

```
{
    "0": {
        // object 1
    },
    "1": {
        // object 2
    },
    "2": {
        // object 3
    }
}
```

So I get a property key for each item. What I want it's something like:

```
{
    "data": [
        {
            // object 1
        },
        {
            // object 2
        },
        {
            // object 3
        }
    ]
}
```

Reply:
Fix with 0c568e7